modules/ossm-federation-config-smcp.adoc

```diff
@@ -0,0 +1,397 @@
+|Port `15443` is required for sending TLS service requests to other meshes in the federation. Port `8188` is required for sending service discovery requests to other meshes in the federation.
+|
+
+|spec:
+  gateways:
+    additionalIngress:
+|Define an additional ingress gateway gateway for _each_ mesh peer in the federation.
+|
+|
+
+|spec:
+  gateways:
+    additionalIgress:
+      <ingressName>:
+        enabled:
+|This parameter enables or disables the federation ingress.
+|`true`/`false`
+|`true`
+
+|spec:
+  gateways:
+    additionalIngress:
+      <ingressName>:
+        router mode:
+|
+|`sni-dnat`
+|
+
+|spec:
+  gateways:
+    additionalIngress:
+      <ingressName>:
+        service:
+          type:
+|The ingress gateway service must be exposed through a load balancer that operates at Layer4 of the OSI model and is publicly available.
+|`LoadBalancer`
+|
+
+|spec:
+  gateways:
+    additionalIngress:
+      <ingressName>:
+        service:
+          metadata:
+            labels:
+              federation.maistra.io/proxy:
+|Specify a unique label for the gateway to prevent federated traffic from flowing through the cluster's default system gateways.
+|
+|
+
+|spec:
+  gateways:
+    additionalIngress:
+      <ingressName>:
+        service:
+          ports:
+|Used to specify the `port:` and `name:` used for TLS and service discovery. Federation traffic consists of raw encrypted TCP for service traffic. Federation traffic consists of HTTPS for discovery.
+|Port `15443` is required for receiving TLS service requests to other meshes in the federation. Port `8188` is required for receiving service discovery requests to other meshes in the federation.
+|
+|===
+
+
+== Understanding federation trust domain parameters
+
+Each mesh in the federation must have its own unique trust domain. This value is used when configuring mesh federation in the `ServiceMeshPeer` resource.
+
+[source,yaml]
+----
+kind: ServiceMeshControlPlane
+metadata:
+  name: red-mesh
+  namespace: red-mesh-system
+spec:
+  security:
+    trust:
+      domain: red-mesh.local
+----
+
+.Federation security parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+|spec:
+  security:
+    trust:
+      domain:
+|Used to specify a unique name for the trust domain for the mesh. Domains must be unique for every mesh in the federation.
+|`<mesh-name>.local`
+|N/A
+|===
+
+
+== Understanding federation DNS parameters
+
+The DNS parameters are configured automatically in the version 2.1 SMCP; you do not need to configure them. They are presented here for reference purposes.
+
+.Federation DNS parameters
+[source,yaml]
+----
+kind: ServiceMeshControlPlane
+metadata:
+  name: red-mesh
+  namespace: red-mesh-system
+spec:
+  techPreview:
+    meshConfig:
+      defaultConfig:
+        proxyMetadata:
+          ISTIO_META_DNS_CAPTURE: "true"
+          PROXY_XDS_VIA_AGENT: "true"
+----
```

Comment:
All of the above `techPreview` settings are not required and are configured by default.

Reply:
Yes, and later in the file I point out that those lines are configured by default.  Question, if they're not required, then it won't matter if customers deleted those lines?  My worry was that if we didn't document them, that someone might delete that configuration and it might break something.

Reply:
They are required and the only way they can be disabled is if they add the block above, but change the settings to false.  I consider these internal details, which users shouldn't have to worry about.  All they need to know is that DNS requests are managed by the proxy.

Reply:
As long as you think customers can't hurt themselves, then should I remove this entire heading and section?

Reply:
Yes, delete the whole DNS section.

Reply:
Deleted DNS section.